docs/README.md

```diff
@@ -0,0 +1,7 @@
+# Building the Ansible AWX Operator Docs
+
+To make a local copy of the AWX Operator docs:
+
+1. cd to the /docs directory. 
+2. pip install --user -r requirements.txt
+3. mkdocs build
```

Comment:
The mkdocs.yml current lives at the root of the checkout, but this command expects the user to run `mkdocs build` in the docs/ directory, which will error because mkdocs.yml isn't there.  

Also, I get this error when running from the root of the checkout:

```
$ pwd
/home/chadams/awx-operator
[chadams@fedora awx-operator]$ mkdocs build
WARNING  -  Config value 'exclude_docs': Unrecognised configuration name: exclude_docs
Aborted with 1 Configuration Warnings in 'strict' mode!
```

Reply:
the mkdocs warning is fixed by upgrading to the most recent mkdocs. I'll update my PR with your other comments, thanks!